base/settings/R/check.all.settings.R

```diff
@@ -96,42 +96,53 @@ check.inputs <- function(settings) {
 ##' @param settings settings file
 ##' @export check.database
 check.database <- function(database) {
-  if (is.null(database)) return(NULL);
+  if (is.null(database)) return(NULL)
   
   ## check database settings
   if (is.null(database$driver)) {
     database$driver <- "PostgreSQL"
-    PEcAn.logger::logger.warn("Please specify a database driver; using default 'PostgreSQL'")
+    PEcAn.logger::logger.info("Database driver unspecified. ",
+                              "Using 'PostgreSQL' (default)")
   }
-  
-  # Attempt to load the driver
-  if (!require(paste0("R", database$driver), character.only=TRUE)) {
-    PEcAn.logger::logger.warn("Could not load the database driver", paste0("R", database$driver))
+
+  is_postgres_like <- database$driver %in% c("PostgreSQL", "Postgres")
+  is_postgresql <- database$driver == "PostgreSQL"
+
+  if (!is_postgres_like) {
+    PEcAn.logger::logger.severe(
+      "Database driver `", database$driver, "` is not supported. ",
+      "Please use `PostgreSQL` or `Postgres`."
+    )
+  }
+
+  if (database$driver == "Postgres") {
+    PEcAn.logger::logger.warn(
+      "Support for RPostgres is experimental. ",
+      "Use at your own risk!."
+    )
   }
   
-  # MySQL specific checks
-  if (database$driver == "MySQL") {
-    if (!is.null(database$passwd)) {
-      PEcAn.logger::logger.info("passwd in database section should be password for MySQL")
-      database$password <- database$passwd
-      database$passwd <- NULL
-    }
-    if (!is.null(database$name)) {
-      PEcAn.logger::logger.info("name in database section should be dbname for MySQL")
-      database$dbname <- database$name
-      database$name <- NULL
-    }
+  # Attempt to load the driver
+  rdriver <- paste0("R", database$driver)
+  if (!requireNamespace(rdriver, quietly = TRUE)) {
+    PEcAn.logger::logger.severe("Could not load the database driver: ", rdriver)
   }
   
   # PostgreSQL specific checks
-  if (database$driver == "PostgreSQL") {
+  if (is_postgres_like) {
     if (!is.null(database$passwd)) {
-      PEcAn.logger::logger.info("passwd in database section should be password for PostgreSQL")
+      PEcAn.logger::logger.warn(
+        "Database field `passwd` is soft-deprecated. ",
+        "Please use `password` instead."
+      )
       database$password <- database$passwd
       database$passwd <- NULL
     }
     if (!is.null(database$name)) {
-      PEcAn.logger::logger.info("name in database section should be dbname for PostgreSQL")
+      PEcAn.logger::logger.warn(
+        "Database field `name` is soft-deprecated. ",
```

Comment:
This may be a personal bugbear, but I don't think 'soft-deprecated' is a useful category. Can we drop the "soft-"?

Reply:
Addressed.